internal/template/templates/.claude/rules/moai/core/settings-management.md

```
@@ -31,7 +31,36 @@ Standard MCP servers in MoAI-ADK:
 - pencil: .pen file design editing. Used by expert-frontend (sub-agent mode) and team-designer (team mode).
 - claude-in-chrome: Browser automation
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)